### PR TITLE
BUGFIX: Fix output of request and response in throwable storage

### PR DESCRIPTION
**Pull request by daniellienert**
Fixed the checking for valid request and response objects.
Remove superflous configuration of throwablestorage in `Scripts::initializeExceptionStorage`

### CODE REVIEW

File: Neos.Flow/Classes/Core/Booting/Scripts.php [hunk 1]
```diff
@@ -263,36 +261,7 @@ protected static function initializeExceptionStorage(Bootstrap $bootstrap): Thro
...
-            $response = $requestHandler->getHttpResponse();
-            // TODO: Sensible error output
-            $output .= PHP_EOL . 'HTTP REQUEST:' . PHP_EOL . ($request ? '[request was empty]' : RequestInformationHelper::renderRequestHeaders($request)) . PHP_EOL;
-            $output .= PHP_EOL . 'HTTP RESPONSE:' . PHP_EOL . ($response ? '[response was empty]' : $response->getStatusCode()) . PHP_EOL;
-            $output .= PHP_EOL . 'PHP PROCESS:' . PHP_EOL . 'Inode: ' . getmyinode() . PHP_EOL . 'PID: ' . getmypid() . PHP_EOL . 'UID: ' . getmyuid() . PHP_EOL . 'GID: ' . getmygid() . PHP_EOL . 'User: ' . get_current_user() . PHP_EOL;
-
-            return $output;
-        });
-
-        return $throwableStorage;
```
Comment (albe) [inline Neos.Flow/Classes/Core/Booting/Scripts.php]: 😱 all that is totally unnecessary?

Comment (daniellienert) [reply Neos.Flow/Classes/Core/Booting/Scripts.php]: Its exactly the same code as in the constructor

**Comment by kdambekalns** [comment]
Just wanted to express some post-merge ❤️!

**Comment by kitsunet** [comment]
Ehhhh, wait, wait, wait! That duplication had a reason :D The FileStorage is just _our_ implementation of the `\Neos\Flow\Log\ThrowableStorageInterface` which has the `setBacktraceRenderer` method. So Flow _should_ set the renderer on initialising the choosen implementation (which might or might not be the FileStorage), BUT given that there might be exceptions BEFORE the init code in Scripts is run the FileStorage would automatically be used as "fallback" exception storage and for that case it had the duplicated code in the constructor. If you have a better idea to solve those cases, feel free, but htis is not ;)

**Comment by kitsunet** [comment]
Please now vote to revert, thanks! 😆

**Comment by albe** [comment]
Oh my, definitely something we should inline document in that duplication. Let's revert quickly then and tag a new release - haven't announced my releases from yesterday yet.

**Pull request merged**